### ISSUE TITLE
Entire PC bluescreens upon connecting wiimote.

### ISSUE THREAD

**Issue by mav1o1** [TheNathannator/WiitarThing#9]
Uploading 20230317_181132.mp4…

I have reinstalled vigembus multiple times, and restarted my pc, but the error still persists.

**Comment by TheNathannator** [comment]
Well, that's not good lol

I really hope it's not some more memory corruption stuff I need to look into, that's gonna be painful to work out. I had to do that for the most recent couple versions, but I already had a suspicion on what was happening that time. Here I've got no ideas.

**Comment by mav1o1** [comment]
Here's a picture of the error it throws up before I blue screens. It was very hard to take a picture of, as it only shows up for around half a second before the PC crashes, but it might help.

**Comment by mav1o1** [comment]
![20230317_194334_jpg_75](https://user-images.githubusercontent.com/62120245/226070751-f213b671-9c8a-4bb7-ae9f-29f2abc329ac.jpg)

**Comment by TheNathannator** [comment]
That error seems rather nonsense, unfortunately. Can you upload the WiitarLog.txt file inside your WiitarThing folder? Assuming it didn't get corrupted from the crash...

I'll see if there's anything I can do to test for memory corruption. I may also have to get in touch with the ViGEmBus developer about the bluescreen, so I'll need the memory dump file Windows created:

1. Go to `C:\Windows` and *copy* the `MEMORY.DMP` file to your desktop (do not move it, you will get file permission issues otherwise)
2. Use 7-Zip to compress it: right-click the dump, go to 7-Zip > `Add to archive...`, set the compression level to Ultra, and start it. It will take a while to finish.
3. Once it's done, upload the compressed file to something like Google Drive and post the link here. I'll let you know when I've grabbed it so you can delete it.

**Comment by TheNathannator** [comment]
Alright, downloaded it. I figured the log file would be blank, very high chance of corruption when a file gets written very shortly before a crash lol

When I get a chance I'll investigate further, thanks for posting an issue!

**Comment by TheNathannator** [comment]
I finally got around to checking the crash dump, it's definitely ViGEmBus at fault here but the error code is different than what I see in the video. I definitely suspect some more memory corruption, if it were an actual bug I'd assume it would be the same code.

Aside from that though, it may be worth it to try reinstalling ViGEmBus if you haven't yet. At some point before I got those initial memory corruption issues fixed, I started getting ViGEmBus crashes of a similar nature which were fixed after a reinstall. Dunno what made it happen in the first place lol

**Comment by mav1o1** [comment]
Hey, just checking in: the blue screen also occurs when using x360ce, a separate ViGEmBus-based application, so it's definitely something related to that.

**Comment by TheNathannator** [comment]
Yeah, definitely sounds like you should try reinstalling ViGEmBus.

I still haven't gotten to doing any memory corruption tests yet, but since it's occuring with other programs I'm more doubtful it's an issue on WiitarThing's side. I still want to do that though, who knows what other issues are hiding in this codebase...

**Comment by scottysace** [comment]
i have the crash without the bluescreen when pressing sync 
i created a seperate user for my pc and it doesnt present the error on there which is weird

**Comment by scottysace** [comment]
i found out what was causing it something to do with my color profile set heres the fix https://www.pingman.com/kb/article/pingplotter-is-giving-an-error-for-systemwindowsbaml2006typeconvertermarkupextension-how-do-i-fix-it-153.html

**Comment by TheNathannator** [comment]
I thought I fixed that bug already lol, I probably need to make a similar fix somewhere in the sync menu.
